src/main/java/io/seqera/tower/cli/commands/pipelines/ImportCmd.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.commands.pipelines;
+
+import io.seqera.tower.ApiException;
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.cli.responses.pipelines.PipelinesCreated;
+import io.seqera.tower.cli.utils.FilesHelper;
+import io.seqera.tower.model.ComputeEnv;
+import io.seqera.tower.model.CreatePipelineRequest;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import static io.seqera.tower.cli.utils.JsonHelper.parseJson;
+
+@CommandLine.Command(
+        name = "import",
+        description = "Create a workspace pipeline from file content"
+)
+public class ImportCmd extends AbstractPipelinesCmd {
+
+    @CommandLine.Option(names = {"-n", "--name"}, description = "Pipeline name", required = true)
+    public String name;
+
+    @CommandLine.Option(names = {"-c", "--compute-env"}, description = "Compute environment name (defaults to json file defined environment)")
+    public String computeEnv;
+
+    @CommandLine.Parameters(index = "0", paramLabel = "FILENAME", description = "File name to import", arity = "1")
+    Path fileName = null;
+
+    @Override
+    protected Response exec() throws ApiException, IOException {
+        CreatePipelineRequest request;
+
+        request = parseJson(FilesHelper.readString(fileName), CreatePipelineRequest.class);
+        request.setName(name);
+
+        if (computeEnv != null) {
+            ComputeEnv ce = findComputeEnvironmentByName(computeEnv, workspaceId());
+            request.getLaunch().setComputeEnvId(ce.getId());
+        }
+
```

Comment:
When no `computeEnvId` is provided at the `json` neither as a command line option, the API returns a http 403 error and then the CLI shows a cryptic error message 
```
ERROR: Unknown. Check that the provided identifier is correct.
```

May be, here we can check that the CE is provided and fail with a nice error message if not.

Reply:
As far as I can see, a CE will always be available in the import `json` file unless y manually remove it, right? Will this scenario ever occur?

Reply:
For me makes sense to manually remove the CE from the `json` if you plan to import that pipeline in many different workspaces.

Reply:
in fact I've also seen that when you try to import a pipeline with a CE from another workspace you also get the same cryptic error:
```
ERROR: Unknown. Check that the provided identifier is correct.
```
So we need to improve this message checking if the user has access to that CE, or opening an issue at Tower repo if you thing that this has to be improved a API side.